lib/guardian.rb

```diff
@@ -129,6 +129,12 @@ def can_moderate?(obj)
   alias :can_see_flags? :can_moderate?
   alias :can_close? :can_moderate?
 
+  def can_tag?(obj)
+    return false unless obj && obj.is_a?(Topic)
```

Comment:
Hmm why do we have to check whether the object is a `Topic`? The only place using this method is always going to pass it an instance of `Topic.`